project_scrum/project_scrum_demo.xml

```diff
@@ -0,0 +1,253 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data noupdate="1">
+
+<!-- Users -->
+        <record id="base.user_demo" model="res.users">
```

Comment:
Please format the lines to help readability of code (usually one attribute per line)